modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -119,6 +136,10 @@ def tracked_item_id
         params[:trackedItemId] || params[:tracked_item_id]
       end
 
+      def file_name
+        params.require(:file_name)
```

Comment:
This requires the file_name param but there’s no explicit rescue for missing parameter (would raise ActionController::ParameterMissing).

Reply:
That is correct, it would raise an error for ActionController::ParameterMissing. Are you suggesting capturing this and raising a Common::Exceptions::ParameterMissing instead?